Backend/app/rider-platform/rider-app/Main/src/Storage/Queries/Person.hs

```diff
@@ -439,5 +462,7 @@ instance ToTType' BeamP.Person Person where
         BeamP.useFakeOtp = useFakeOtp,
         BeamP.followsRide = followsRide,
         BeamP.falseSafetyAlarmCount = Just falseSafetyAlarmCount,
-        BeamP.safetyCenterDisabledOnDate = safetyCenterDisabledOnDate
+        BeamP.safetyCenterDisabledOnDate = safetyCenterDisabledOnDate,
+        BeamP.referredByCustomer = referredByCustomer,
+        BeamP.customerReferralCode = customerReferralCode
```

Comment:
only referralCode name should be fine, its customer side...customer prefix is not required.